src/sign-in-webapp/README.md

```diff
@@ -67,46 +70,143 @@ Later the web app can make calls to the Microsoft 365 Graph API on behalf of sig
    az ad app update --id $AZURE_AD_APP_CLIENT_ID_WEBAPP --set logoutUrl="https://localhost:5001/signout-oidc"
    ```
 
-## Scaffold the web app by using the ASP.NET Core Razor project template
+</details>
 
-1. Install the latest `Microsoft.Identity.Web` templates from NuGet.org
+### 2. Configure the web app
 
-   ```bash
-   dotnet new --install Microsoft.Identity.Web.ProjectTemplates::1.21.0
-   ```
+1. Open the `WebApp.csrpoj` under the the `sign-in-webapp` folder in your code editor.
+1. Open the `appsettings.json` file and modify the following code:
 
-1. Create a Microsoft identity platform (Azure AD v2.0) application. Execute the following command to create the web app project
+    ```json
+    "Domain": "[Enter the domain of your tenant, e.g. contoso.onmicrosoft.com]",
+    "TenantId": "[Enter 'common', or 'organizations' or the Tenant ID (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+    "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
+    "ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
+    ```
 
-   ```bash
-   dotnet new webapp2 --auth SingleOrg --aad-instance "https://login.microsoftonline.com/" --client-id ${AZURE_AD_APP_CLIENT_ID_WEBAPP} --tenant-id $(az account show --query tenantId --output tsv) --domain ${AZURE_AD_APP_DOMAIN} --called-api-url "https://graph.microsoft.com/v1.0/me"
-   ```
+<details>
+   <summary>:computer: Alternative: modify the `appsettings.json` file from your terminal</summary>
 
-   :warning: Today, with ASP.NET Core 3.1 without Microsoft Identity Web, when running `dotnet new --auth` and creating a `webapp` from an ASP.NET core template, the application is targeting the Azure AD v1.0 endpoint. Microsoft Identity Web provides the glue between the ASP.NET Core middleware and MSAL .NET to bring a clearer, more robust developer experience, which also leverages the power of the Microsoft identity platform (formerly Azure AD v2.0 endpoint). `Microsoft.Identity.Web` is a simpler way to use Azure AD in ASP.NET Core web apps and web APIs. It doesn't replace ASP.NET Identity in any way, it doesn't replace AddOpenIdConnect, AddJwtBearer or AddCookie or any of the lower level primitives, but it does use and configure them correctly for Azure AD. It replaces AzureAD.UI and AzureADB2C.UI which are obsolete in .NET 5.0 or later. For more information, please take a look at https://github.com/AzureAD/microsoft-identity-web/wiki/Microsoft-Identity-Web-basics#why-use-microsoftidentityweb
+1. Create the `appsettings.json` file with the Azure AD app configuration
 
-## Configure the web app
+   ```bash
+   cat > appsettings.json <<EOF
```

Comment:
This, however, might be a problem. I'm worried that with code in the README that won't be built/tested, we won't know if it breaks if/when there's a breaking change to the libraries, framework, etc.

Reply:
do you want us to remove this or wait and see?

Reply:
Let's go ahead and remove the alternative path.

Reply:
done | addressed from [3a240](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/14/commits/3a240092dc851a96215b3513b8d027112ccf2762)

Reply:
Should the alt path also include the 

```
### 3. Clean up
1. Delete the Azure AD app
```

part?

Reply:
thanks for posting this out @ckittel 

done | addressed [048e4](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/14/commits/048e459325d3c0c696ecec2fd05f8c70f1fa2e6f)

Reply:
Gone!  Thanks.